trace/opencensus/trace/tracer/context_tracer.py

```diff
@@ -78,6 +78,21 @@ def set_enabled(self):
         else:
             return False
 
+    def trace_decorator(self):
+        """Decorator to trace a function."""
+
+        def decorator(func):
+
+            def wrapper(*args, **kwargs):
```

Comment:
To me it's reasonable to trace the function for every execution in a context. Because it might belong to different parent span, and worth to know the latency of each execution. And the `start_span` and `end_span` method in the wrapper has the information in context tracer and span context, like whether this function should be sampled and what's the parent span.